R/run-phantomjs.R

```diff
@@ -39,15 +39,19 @@ run_phantomjs <- function(debugLevel = c("INFO", "ERROR", "WARN", "DEBUG"),
   # The env vars are a workaround for :
   # https://github.com/rstudio/shinytest/issues/165#issuecomment-364935112
   withr::with_envvar(get_phantom_envvars(), {
-    ph <- process$new(command = phexe, args = args, supervise = TRUE,
-                      stdout = tempfile("webdriver-stdout-", fileext = ".log"),
-                      stderr = tempfile("webdriver-stderr-", fileext = ".log"))
+    ph <- process$new(
+      command = phexe,
+      args = args,
+      supervise = TRUE,
+      stdout = tempfile("webdriver-stdout-", fileext = ".log"),
+      stderr = "2>&1"
```

Comment:
Are there any practical drawbacks to combining the stdout and stderr streams?

Reply:
I don't think so. I think it is pretty rare that you need them separate. E.g. if you are in a terminal, then they are mixed as well, and all is fine.